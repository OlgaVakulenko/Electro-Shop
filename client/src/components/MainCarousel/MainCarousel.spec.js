import React from 'react';
import { shallow } from 'enzyme';

import { MainCarousel } from './MainCarousel';

describe('The test of MainCarousel component', () => {
  it('MainCarousel was rendered', () => {
    const rendered = shallow(
      <MainCarousel />,
    );

    expect(rendered).toMatchSnapshot();
  });
});
