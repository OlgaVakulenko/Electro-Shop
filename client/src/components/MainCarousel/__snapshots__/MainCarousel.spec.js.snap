// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The test of MainCarousel component MainCarousel was rendered 1`] = `<Preloader />`;
